keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -198,11 +195,11 @@ which go beyond running particular images.
 
 - This proposal does not aim to replace existing `VolumeSource` types.
 - This proposal does not address other use cases for OCI objects beyond directory sharing among containers in a pod.
-- Mounting thousands of images and artifacts in a single pod.
+- Mounting thousands of images in a single pod.
 - The enhancement leaves single file use case out for now and restrict the mount
   output to directories.
 - The runtimes (CRI-O, containerd, others) will have to agree on the
-  implementation of how artifacts are manifested as directories. We don't want
+  implementation of how images are manifested as directories. We don't want
```

Comment:
This bullet was tied to the issue of what is an artifact how is it manifested as directories on a per artifact type basis.. the layering of images by contrast are well known. So this bullet does not make sense when changing it to image.

Reply:
Added a note that it's out of scope.